certbot-ci/certbot_integration_tests/utils/acme_server.py

```diff
@@ -188,11 +195,12 @@ def _prepare_http_proxy(self):
         self._launch_process(command)
         print('=> Finished configuring the HTTP proxy.')
 
-    def _launch_process(self, command, cwd=os.getcwd(), env=None):
+    def _launch_process(self, command, cwd=os.getcwd(), env=None, force_output=False):
         """Launch silently a subprocess OS command"""
         if not env:
             env = os.environ
-        process = subprocess.Popen(command, stdout=self._stdout, stderr=subprocess.STDOUT, cwd=cwd, env=env)
+        stdout = sys.stdout if force_output else self._stdout
```

Comment:
It is not a very good idea to dump data to stdout in the context of pytest-xdist, since it is used to coordinate the parallel workers in that context.

Since it is for debugging purpose, and stderr is left available by pytest-xdist for that matter, could we just dump to stderr here?

Reply:
If I make that change, I'm imagining something like changing the name of the parameter here to `force_stderr` and this line to:
```
        stdout = sys.stderr if force_stderr else self._stdout
```
and leaving everything else the same.

That work for you?

Reply:
Totally!